cli/cmd/compliance_aws.go

```diff
@@ -251,6 +338,10 @@ func init() {
 	complianceAwsCmd.AddCommand(complianceAwsListAccountsCmd)
 	complianceAwsCmd.AddCommand(complianceAwsRunAssessmentCmd)
 
+	// Experimental Commands
```

Comment:
These commands are marked as experimental (in this comment) but not hidden from help messages.

Reply:
👍  will mark as hidden